pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -27,16 +27,8 @@ def work(self):
         total_bag_space = self.bot.player_data['max_item_storage']
         free_bag_space = total_bag_space - items_in_bag
 
-        if self.min_empty_space is not None:
-            if free_bag_space >= self.min_empty_space:
-                    self.emit_event(
-                        'item_discard_skipped',
-                        formatted="Skipping Recycling of Items. {space} space left in bag.",
-                        data={
-                            'space': free_bag_space
-                        }
-                    )
-                    return
+        if self.min_empty_space is not None and free_bag_space >= self.min_empty_space:
```

Comment:
Might be worth switching this to if `isinstance(self.min_empty_space, int)` in the interest of also not returning when people accidentally break the syntax of config files...